src/ResourceManager/Insights/Commands.Insights/Alerts/RemoveAzureRmAlertRuleCommand.cs

```diff
@@ -49,8 +50,8 @@ public class RemoveAzureRmAlertRuleCommand : ManagementCmdletBase
         /// </summary>
         protected override void ProcessRecordInternal()
         {
-            AzureOperationResponse result = this.InsightsManagementClient.AlertOperations.DeleteRuleAsync(resourceGroupName: this.ResourceGroup, ruleName: this.Name).Result;
-            WriteObject(result);
+            this.InsightsManagementClient.AlertRules.DeleteAsync(resourceGroupName: this.ResourceGroup, ruleName: this.Name).Wait();
+            WriteObject("");
```

Comment:
@gucalder why was the output type of the cmdlet changed so that it could return nothing?